Minutes of Jan 30th 2025 coordination meeting (#1657)
HSF/hsf.github.io: organization/_posts/2025/2025-01-30-coordination.md

---
title: "HSF Coordination Meeting #281, 30 January 2025"
layout: plain_toc
---

## Attending

Present/Contributing: Graeme Stewart, Matthew Feickert, Luke Kreczko, Eduardo Rodrigues, Alex Held, Paul Laycock, Peter Fackeldey, Tommaso Lari, Juan Migel Carceller, Ianna Osborne, Davide Costanzo, Ruslan Mashinistov, Krzysztof Genser, Joe Osborn, Richa Sharma, Saptaparna Bhattacharya, Patrick Gartung, Uwe Hernandez Acosta, Nick Smith, Alexander Moreno, Claire Antel, Liz Sexton-Kennedy, Steven Gardiner, Christian Wessel

Apologies/Contributing: Michel Jouvin, Michel Hernandez Villanueva

## News, general matters, announcements

### European Strategy Update (ESPPU)

We now have an [advanced draft](https://docs.google.com/document/d/1dTUBTlerXHjrKO37hAuK1MednfukOxvkCSnq0HrnPOM/edit?usp=sharing) - thanks to everyone who contributed to get the document to this stage.

> Last week we got the inputs from the activity areas and Michel and Graeme did an editing pass, harmionising the style and drawing out key points as bulleted recommendations (for the TL;DR crew). A preamble and conclusion have been added.

Reviewer: ```suggestion
Last week we got the inputs from the activity areas and Michel and Graeme did an editing pass, harmonising the style and drawing out key points as bulleted recommendations (for the TL;DR crew). A preamble and conclusion have been added.
```

> The section on Event Generators still needs some more key input re. FCCee in particular; the other sections are close to final for this stage.

Reviewer: ```suggestion
The section on Event Generators still needs some more key input re. FCC-ee in particular; the other sections are close to final for this stage.
```

    - Thanks to Phil Ilten who steps down
- PyHEP: Nikolai Hartmann (continues), Ianna Osborne (new), Peter Fackeldey (new), Matthew Feickert (stepping down but continues as advisory and to help with PyHEP)
    - Thanks to Eduardo Rodrigues and Jim Pivarski who step down
- Reconstruction and SW Triggers: Inês Ochoa (new), Joe Osborn (continues), Maarten Van Veghel (new), Christian Wessel (new)
    - Thanks to Claire Antel and Christina Agapopoulou who step down
- Development Tools and Packaging: Ruslan Mashinistov (new), Juan Migel Carceller (new), Wouter Deconinck (new)
    - Thanks to Andrea Valenzuela Ramirez, Patrick Gartung, Reiner Hauser who step down
- Training: Michel Hernandez Villanueva (new), Richa Sharma (new), Alexander Moreno (continues), Lera Lukashenko (continues)
    - Thanks to Holly Szumila-Vance and Jim Pivarski
- JuliaHEP: Pere Mato (continues), Jerry Ling (new), Alexander Moreno (continues), Uwe Hernandez Acosta (continues)
    - Thanks to Tamas Gal who steps down

Those conveners who are stepping down will be relieved of being on the mailing list after this meeting.

> N.B. To better reflect our new nomenclature, we propose to rename the mailing list from `hsf-wg-convenors@googlegroups.com` to `hsf-activity-convenors@googlegroups.com`

Reviewer: ```suggestion
N.B. To better reflect our new nomenclature, we propose to rename the mailing list from `hsf-wg-convenors@googlegroups.com` to `hsf-activity-conveners@googlegroups.com`
```

Reviewer: (Better now use the name of the list we converged on.)
